github_actions/lib/dependabot/github_actions.rb

```diff
@@ -15,6 +15,7 @@
   register_label_details(
     "github_actions",
     name: "github_actions",
+    description: "Pull requests that update GitHub action code",
```

Comment:
```suggestion
    description: "Pull requests that update GitHub Actions code",
```

Should we use "GitHub Actions" instead? That seems to be the official way they are referred to: https://docs.github.com/en/actions